docs/guides/extension_points.rst

```diff
@@ -0,0 +1,165 @@
+| Learning Context    | ``openedx.learning_context``             | A "Learning Context" is a course, a library, a program, a   |
+|                     |                                          | blog, an external site, or some other collection of content |
+|                     |                                          | where learning happens. If you are trying to build a        |
+|                     |                                          | totally new learning experience that's not a type of course,|
+|                     |                                          | you may need to implement a new learning context.           |
+|                     |                                          | Learning contexts are a new abstraction and are only        |
+|                     |                                          | supported in the nascent Blockstore-based XBlock runtime.   |
+|                     |                                          | Since existing courses use modulestore instead of           |
+|                     |                                          | Blockstore, they are not yet implemented as learning        |
+|                     |                                          | contexts. However, Blockstore-based content libraries are.  |
+|                     |                                          | See |learning_context.py|_ to learn more.                   |
++---------------------+------------------------------------------+-------------------------------------------------------------+
+| User partition      | ``openedx.user_partition_scheme`` and    | A user partition scheme is a named way for dividing users   |
+| scheme              | ``openedx.dynamic_partition_generator``  | in a course into groups, usually to show different content  |
+|                     |                                          | to different users or to run experiments. Partitions may be |
+|                     |                                          | added to a course manually, or automatically added by a     |
+|                     |                                          | "dynamic partition generator." The core platform includes   |
+|                     |                                          | partition scheme plugins like ``random``, ``cohort``,       |
+|                     |                                          | and ``enrollment_track``. See the |UserPartition docstring|_|
+|                     |                                          | to learn more.                                              |
++---------------------+------------------------------------------+-------------------------------------------------------------+
+| XBlock              | ``xblock.v1``                            | An XBlock provides a new type of learnable content that can |
+|                     |                                          | be used in courses, content libraries, etc. See "Content    |
```

Comment:
nit: please link to [Content Integrations](#content-integrations) anchor.

Reply:
Ok, done.